app/Http/Controllers/MessagesController.php

```diff
@@ -30,12 +30,12 @@ public function index(){
         return view('messenger.index', compact('threads', 'currentUserId', 'users'));
     }
 
-    public function create(){
+    public function create() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->